### PR TITLE
[#126584725] fix(display): fix display issues

### PR DESCRIPTION
**Pull request by andela-ajamiu**
#[126584725]

#### What does this PR do?
It fixes the display issues on certain views.

### CODE REVIEW

File: spec/controllers/subscriptions_controller_spec.rb [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-    end
-
-    context "when event manager updates an event" do
-      it "should diliver email to subscribers" do
-        put "/events/#{@event.id}", attributes_for(:regular_event)
-        expect(ActionMailer::Base.deliveries.size).to eq 1
-      end
-    end
-  end
-end
```
Comment (andela-megwali) [inline spec/controllers/subscriptions_controller_spec.rb]: Are these tests obsolete?

File: spec/factories/events.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
     title "Blessings wedding"
     description "Happy day of joy celebration happinness smiles."
     location "Beach side"
-    start_date Time.zone.now + 86_400
```
Comment (andela-megwali) [inline spec/factories/events.rb]: I thought we agreed to keep this?

File: spec/factories/subscription.rb [hunk 3]
```diff
@@ -1,7 +0,0 @@
-FactoryGirl.define do
-  factory :subscription do
-    event_id 1
-    manager_profile_id 1
-    user
-  end
-end
```
Comment (andela-megwali) [inline spec/factories/subscription.rb]: Is subscriptions no longer a required feature?

File: spec/features/attending_events_spec.rb [hunk 4]
```diff
@@ -31,16 +31,16 @@
 
   scenario "User tries to attend Event" do
     visit events_path
-    find_link("Blessings wedding").trigger("click")
+    click_link "Blessings wedding"
```
Comment (andela-megwali) [inline spec/features/attending_events_spec.rb]: I thought find_link was more reliable?

File: spec/features/subscription_spec.rb [hunk 5]
```diff
@@ -1,35 +0,0 @@
...
-      event_id: @event.id
-    )
-    visit "/events/#{@event.id}"
-
-    find_button("unsubscribeBtn").trigger("click")
-    page.evaluate_script("window.confirm = function() { return true; }")
-
-    expect(page).to have_content "You have unsubscribed from this event"
-  end
-end
```
Comment (andela-megwali) [inline spec/features/subscription_spec.rb]: Are these obsolete?

**Comment by andela-ajamiu** [comment]
A different PR has been raised to address the issue.

**Pull request closed without merging**